server/webapp/WEB-INF/rails.new/webpack/views/dashboard/trigger_with_options/modal_body.js.msx

```diff
@@ -37,11 +38,15 @@ const ModalBody = {
   view(vnode) {
     let secureVarsTabContent, environmentVarsTabContent, secureVarsTab, environmentVarsTab;
 
-    const self = vnode.state;
-    const vm   = vnode.attrs.vm();
-    const info = vnode.attrs.triggerWithOptionsInfo;
+    const self    = vnode.state;
+    const vm      = vnode.attrs.vm();
+    const info    = vnode.attrs.triggerWithOptionsInfo;
+    const message = vnode.attrs.message;
 
     if (!info()) {
+      if (message()) {
+        return (<f.alert>{message()}</f.alert>);
```

Comment:
This seems to imply that the message will always be an error. Can we rename the `message` attribute to be `errorMessage` instead to clarify intent?

Reply:
Actually, ignore my comment, not worth it, since it's used in only one place.